diff --git a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java
index b02f50b..6df349b 100644
--- a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java
+++ b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java
@@ -82,4 +82,10 @@ public class ARTRenderableViewManager extends ViewManager<View, ReactShadowNode>
   public void updateExtraData(View root, Object extraData) {
     throw new IllegalStateException("ARTShape does not map into a native view");
   }
+
+  @Override
+  public View prepareToRecycleView(ThemedReactContext reactContext, View view) {
+    // No-op for ART, since it doesn't actually render native views.
+    return view;
+  }
 }
\ No newline at end of file
diff --git a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java
index cf57bbb..c56dfd9 100644
--- a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java
+++ b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java
@@ -22,7 +22,7 @@ import com.facebook.react.common.ReactConstants;
 import com.facebook.react.uimanager.annotations.ReactProp;
 import javax.annotation.Nullable;
 
-import static com.facebook.react.common.ArrayUtils.copyArray;
+import java.util.Arrays;
 
 /**
  * Shadow node for virtual ARTShape view
diff --git a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTVirtualNode.java b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTVirtualNode.java
index d2b4018..9d9904a 100644
--- a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTVirtualNode.java
+++ b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTVirtualNode.java
@@ -11,7 +11,7 @@ import android.graphics.Canvas;
 import android.graphics.Color;
 import android.graphics.Matrix;
 import android.graphics.Paint;
-import android.support.v4.graphics.ColorUtils;
+import androidx.core.graphics.ColorUtils;
 
 import com.facebook.react.bridge.JSApplicationIllegalArgumentException;
 import com.facebook.react.bridge.ReadableArray;
