diff --git a/node_modules/@react-native-community/art/android/build.gradle b/node_modules/@react-native-community/art/android/build.gradle
index 82666ab..a34c5b1 100644
--- a/node_modules/@react-native-community/art/android/build.gradle
+++ b/node_modules/@react-native-community/art/android/build.gradle
@@ -1,14 +1,3 @@
-buildscript {
-  repositories {
-    google()
-    jcenter()
-  }
-
-  dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
-  }
-}
-
 def getExtOrDefault(name) {
   return rootProject.ext.has(name) ? rootProject.ext.get(name) : project.properties['ReactNativeART_' + name]
 }
@@ -31,21 +20,9 @@ android {
 
 repositories {
   google()
-  jcenter()
   mavenCentral()
 }
 
-allprojects {
-    repositories {
-        mavenLocal()
-        google()
-        jcenter()
-        maven {
-            url "$rootDir/../node_modules/react-native/android"
-        }
-    }
-}
-
 dependencies {
     //noinspection GradleDynamicVersion
     api 'com.facebook.react:react-native:+'
diff --git a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java
index b02f50b..2b71f0d 100644
--- a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java
+++ b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTRenderableViewManager.java
@@ -82,4 +82,9 @@ public class ARTRenderableViewManager extends ViewManager<View, ReactShadowNode>
   public void updateExtraData(View root, Object extraData) {
     throw new IllegalStateException("ARTShape does not map into a native view");
   }
+
+  @Override
+  protected View prepareToRecycleView(ThemedReactContext reactContext, View view) {
+    return view;
+  }
 }
diff --git a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java
index cf57bbb..04f0c5f 100644
--- a/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java
+++ b/node_modules/@react-native-community/art/android/src/main/java/com/reactnativecommunity/art/ARTShapeShadowNode.java
@@ -22,8 +22,6 @@ import com.facebook.react.common.ReactConstants;
 import com.facebook.react.uimanager.annotations.ReactProp;
 import javax.annotation.Nullable;
 
-import static com.facebook.react.common.ArrayUtils.copyArray;
-
 /**
  * Shadow node for virtual ARTShape view
  */
