diff --git a/node_modules/@react-native-community/slider/common/cpp/react/renderer/components/RNCSlider/RNCSliderMeasurementsManager.h b/node_modules/@react-native-community/slider/common/cpp/react/renderer/components/RNCSlider/RNCSliderMeasurementsManager.h
index 5d20b2b..17c7ed8 100644
--- a/node_modules/@react-native-community/slider/common/cpp/react/renderer/components/RNCSlider/RNCSliderMeasurementsManager.h
+++ b/node_modules/@react-native-community/slider/common/cpp/react/renderer/components/RNCSlider/RNCSliderMeasurementsManager.h
@@ -10,12 +10,12 @@ namespace facebook::react {
     class RNCSliderMeasurementsManager {
     public:
         RNCSliderMeasurementsManager(
-                const ContextContainer::Shared &contextContainer)
+                const std::shared_ptr<const ContextContainer> &contextContainer)
                 : contextContainer_(contextContainer) {}
 
         Size measure(SurfaceId surfaceId, LayoutConstraints layoutConstraints) const;
 
     private:
-        const ContextContainer::Shared contextContainer_;
+        const std::shared_ptr<const ContextContainer> contextContainer_;
         mutable std::mutex mutex_;
         mutable bool hasBeenMeasured_ = false;
         mutable Size cachedMeasurement_{};
