diff --git a/node_modules/react-native-fetch-blob/android/build.gradle b/node_modules/react-native-fetch-blob/android/build.gradle
index 919e64a..09ff787 100644
--- a/node_modules/react-native-fetch-blob/android/build.gradle
+++ b/node_modules/react-native-fetch-blob/android/build.gradle
@@ -6,15 +6,16 @@ repositories {
 
 buildscript {
     repositories {
-        jcenter()
+        google() // <-- ADD THIS LINE
+        mavenCentral()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:4.2.2'
     }
 }
 
 android {
-    compileSdkVersion 23
+    compileSdkVersion 34
     buildToolsVersion "23.0.1"
     defaultConfig {
         minSdkVersion 16
@@ -33,6 +34,6 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
     //{RNFetchBlob_PRE_0.28_DEPDENDENCY}
 }
diff --git a/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec b/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec
index f702d28..a1d70ec 100644
--- a/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec
+++ b/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec
@@ -9,5 +9,6 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/wkh237/react-native-fetch-blob", :tag => 'v0.10.6'}
   s.source_files = 'ios/**/*.{h,m}'
   s.platform     = :ios, "7.0"
-  s.dependency 'React/Core'
+  s.dependency 'React-Core'
+  s.dependency 'React-RCTBlob'
 end
