diff --git a/node_modules/react-native-fetch-blob/android/build.gradle b/node_modules/react-native-fetch-blob/android/build.gradle
index e4901d6..355a4a6 100644
--- a/node_modules/react-native-fetch-blob/android/build.gradle
+++ b/node_modules/react-native-fetch-blob/android/build.gradle
@@ -1,24 +1,20 @@
 apply plugin: 'com.android.library'
 
+def safeExtGet(prop, fallback) {
+    rootProject.ext.has(prop) ? rootProject.ext.get(prop) : fallback
+}
+
 repositories {
     mavenCentral()
 }
 
-buildscript {
-    repositories {
-        jcenter()
-    }
-    dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
-    }
-}
-
 android {
-    compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    namespace "com.RNFetchBlob"
+    compileSdkVersion safeExtGet('compileSdkVersion', 23)
+
     defaultConfig {
-        minSdkVersion 16
-        targetSdkVersion 23
+        minSdkVersion safeExtGet('minSdkVersion', 16)
+        targetSdkVersion safeExtGet('targetSdkVersion', 23)
         versionCode 1
         versionName "1.0"
     }
@@ -33,6 +29,6 @@ android {
 }
 
 dependencies {
-    compile 'com.facebook.react:react-native:+'
+    implementation 'com.facebook.react:react-native:+'
     //{RNFetchBlob_PRE_0.28_DEPDENDENCY}
 }
diff --git a/node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob/RNFetchBlobReq.java b/node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob/RNFetchBlobReq.java
index 8a81a83..39a9afa 100644
--- a/node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob/RNFetchBlobReq.java
+++ b/node_modules/react-native-fetch-blob/android/src/main/java/com/RNFetchBlob/RNFetchBlobReq.java
@@ -23,7 +23,7 @@ import com.facebook.react.bridge.WritableArray;
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.modules.core.DeviceEventManagerModule;
 import com.facebook.react.modules.network.OkHttpClientProvider;
-import com.facebook.react.modules.network.TLSSocketFactory;
+import javax.net.ssl.SSLSocketFactory;
 
 import java.io.File;
 import java.io.FileOutputStream;
@@ -695,7 +695,7 @@ public class RNFetchBlobReq extends BroadcastReceiver implements Runnable {
     public static OkHttpClient.Builder enableTls12OnPreLollipop(OkHttpClient.Builder client) {
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN && Build.VERSION.SDK_INT <= Build.VERSION_CODES.KITKAT) {
             try {
-                client.sslSocketFactory(new TLSSocketFactory());
+                client.sslSocketFactory((SSLSocketFactory) SSLSocketFactory.getDefault());
 
                 ConnectionSpec cs = new ConnectionSpec.Builder(ConnectionSpec.MODERN_TLS)
                         .tlsVersions(TlsVersion.TLS_1_2)
diff --git a/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec b/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec
index f702d28..c29e11b 100644
--- a/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec
+++ b/node_modules/react-native-fetch-blob/react-native-fetch-blob.podspec
@@ -9,5 +9,5 @@ Pod::Spec.new do |s|
   s.source       = { :git => "https://github.com/wkh237/react-native-fetch-blob", :tag => 'v0.10.6'}
   s.source_files = 'ios/**/*.{h,m}'
   s.platform     = :ios, "7.0"
-  s.dependency 'React/Core'
+  s.dependency 'React-Core'
 end
