diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraView.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraView.kt
index 3a5abae..b48c7be 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraView.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraView.kt
@@ -269,9 +269,8 @@ class CameraView(context: Context, private val frameProcessorThread: ExecutorSer
   private external fun initHybrid(): HybridData
   private external fun frameProcessorCallback(frame: ImageProxy)
 
-  override fun getLifecycle(): Lifecycle {
-    return lifecycleRegistry
-  }
+  override val lifecycle: Lifecycle
+    get() = lifecycleRegistry
 
   /**
    * Updates the custom Lifecycle to match the host activity's lifecycle, and if it's active we narrow it down to the [isActive] and [isAttachedToWindow] fields.
diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraViewManager.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraViewManager.kt
index 35e01e4..ac259da 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraViewManager.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/CameraViewManager.kt
@@ -23,12 +23,12 @@ class CameraViewManager(reactContext: ReactApplicationContext) : ViewGroupManage
   }
 
   override fun getExportedCustomDirectEventTypeConstants(): MutableMap<String, Any>? {
-    return MapBuilder.builder<String, Any>()
-      .put("cameraViewReady", MapBuilder.of("registrationName", "onViewReady"))
-      .put("cameraInitialized", MapBuilder.of("registrationName", "onInitialized"))
-      .put("cameraError", MapBuilder.of("registrationName", "onError"))
-      .put("cameraPerformanceSuggestionAvailable", MapBuilder.of("registrationName", "onFrameProcessorPerformanceSuggestionAvailable"))
-      .build()
+    return mutableMapOf(
+      "cameraViewReady" to MapBuilder.of("registrationName", "onViewReady"),
+      "cameraInitialized" to MapBuilder.of("registrationName", "onInitialized"),
+      "cameraError" to MapBuilder.of("registrationName", "onError"),
+      "cameraPerformanceSuggestionAvailable" to MapBuilder.of("registrationName", "onFrameProcessorPerformanceSuggestionAvailable"),
+    )
   }
 
   override fun getName(): String {
diff --git a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt
index 48b85ca..21930af 100644
--- a/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt
+++ b/node_modules/react-native-vision-camera/android/src/main/java/com/mrousavy/camera/frameprocessor/FrameProcessorRuntimeManager.kt
@@ -5,6 +5,7 @@ import androidx.annotation.Keep
 import com.facebook.jni.HybridData
 import com.facebook.proguard.annotations.DoNotStrip
 import com.facebook.react.bridge.ReactApplicationContext
+import com.facebook.react.common.annotations.FrameworkAPI
 import com.facebook.react.turbomodule.core.CallInvokerHolderImpl
 import com.facebook.react.uimanager.UIManagerHelper
 import com.mrousavy.camera.CameraView
@@ -13,6 +14,7 @@ import java.lang.ref.WeakReference
 import java.util.concurrent.ExecutorService
 
 @Suppress("KotlinJniMissingFunction") // I use fbjni, Android Studio is not smart enough to realize that.
+@OptIn(FrameworkAPI::class)
 class FrameProcessorRuntimeManager(context: ReactApplicationContext, frameProcessorThread: ExecutorService) {
   companion object {
     const val TAG = "FrameProcessorRuntime"
diff --git a/node_modules/react-native-vision-camera/ios/React Utils/JSIUtils.mm b/node_modules/react-native-vision-camera/ios/React Utils/JSIUtils.mm
index adbb762..d35ead9 100644
--- a/node_modules/react-native-vision-camera/ios/React Utils/JSIUtils.mm	
+++ b/node_modules/react-native-vision-camera/ios/React Utils/JSIUtils.mm	
@@ -20,6 +20,7 @@
 #import <ReactCommon/CallInvoker.h>
 #import <React/RCTBridge.h>
 #import <ReactCommon/TurboModuleUtils.h>
+#import <react/bridging/CallbackWrapper.h>
 #import "../Frame Processor/Frame.h"
 #import "../Frame Processor/FrameHostObject.h"
 
