diff --git a/node_modules/react-native-image-marker/android/src/main/java/com/jimmydaddy/imagemarker/ImageMarkerManager.java b/node_modules/react-native-image-marker/android/src/main/java/com/jimmydaddy/imagemarker/ImageMarkerManager.java
index 3e29935..f074e4c 100644
--- a/node_modules/react-native-image-marker/android/src/main/java/com/jimmydaddy/imagemarker/ImageMarkerManager.java
+++ b/node_modules/react-native-image-marker/android/src/main/java/com/jimmydaddy/imagemarker/ImageMarkerManager.java
@@ -105,7 +105,9 @@ public class ImageMarkerManager extends ReactContextBaseJavaModule {
     String minor = getStringSafe("minor", ReactNativeVersion.VERSION);
     String patch = getStringSafe("patch", ReactNativeVersion.VERSION);
     if (Integer.valueOf(major) >= 0 && Integer.valueOf(minor) >= 60 && Integer.valueOf(patch) >= 0) {
-      ImagePipelineConfig config = ImagePipelineConfig.newBuilder(this.context).experiment().setMaxBitmapSize(maxSize).build();
+    ImagePipelineConfig.Builder builder = ImagePipelineConfig.newBuilder(this.context);
+builder.experiment().setMaxBitmapSize(maxSize);
+ImagePipelineConfig config = builder.build();
       Fresco.initialize(this.context, config);
     }
   }
