diff --git a/node_modules/react-native-photo-editor/android/build.gradle b/node_modules/react-native-photo-editor/android/build.gradle
index 603a049..acadcc3 100644
--- a/node_modules/react-native-photo-editor/android/build.gradle
+++ b/node_modules/react-native-photo-editor/android/build.gradle
@@ -44,7 +44,8 @@ repositories {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    implementation 'com.github.prscX:photo-editor-android:master'
+    // implementation 'com.github.prscX:photo-editor-android:master'
+    implementation 'com.github.Rauzon:photo-editor-android:v1.1.0'
     implementation 'fr.avianey.com.viewpagerindicator:library:2.4.1@aar'
     implementation 'com.nineoldandroids:library:2.4.0'
     implementation 'com.android.support:design:28.0.0'
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/053751e7bde870c310a484df3d2b4e72.bin b/node_modules/react-native-photo-editor/android/build/.transforms/053751e7bde870c310a484df3d2b4e72.bin
deleted file mode 100644
index 0d259dd..0000000
--- a/node_modules/react-native-photo-editor/android/build/.transforms/053751e7bde870c310a484df3d2b4e72.bin
+++ /dev/null
@@ -1 +0,0 @@
-o/classes
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/053751e7bde870c310a484df3d2b4e72/classes/classes.dex b/node_modules/react-native-photo-editor/android/build/.transforms/053751e7bde870c310a484df3d2b4e72/classes/classes.dex
deleted file mode 100644
index a545882..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/053751e7bde870c310a484df3d2b4e72/classes/classes.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/1691c5592af3516c0cdef262db98bc39.bin b/node_modules/react-native-photo-editor/android/build/.transforms/1691c5592af3516c0cdef262db98bc39.bin
deleted file mode 100644
index 1ed65e0..0000000
--- a/node_modules/react-native-photo-editor/android/build/.transforms/1691c5592af3516c0cdef262db98bc39.bin
+++ /dev/null
@@ -1 +0,0 @@
-i/
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1.bin b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1.bin
deleted file mode 100644
index 5ff383e..0000000
--- a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1.bin
+++ /dev/null
@@ -1 +0,0 @@
-o/debug
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$1.dex
deleted file mode 100644
index ad5a1e6..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$2.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$2.dex
deleted file mode 100644
index b6ce29a..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$2.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$3.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$3.dex
deleted file mode 100644
index db86e0b..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity$3.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity.dex
deleted file mode 100644
index 1953f1f..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/BaseActivity.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$OnColorPickerClickListener.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$OnColorPickerClickListener.dex
deleted file mode 100644
index c305844..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$OnColorPickerClickListener.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$ViewHolder$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$ViewHolder$1.dex
deleted file mode 100644
index e99ec71..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$ViewHolder$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$ViewHolder.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$ViewHolder.dex
deleted file mode 100644
index 3951154..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter$ViewHolder.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter.dex
deleted file mode 100644
index 707748a..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ColorPickerAdapter.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$OnEmojiClickListener.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$OnEmojiClickListener.dex
deleted file mode 100644
index 34d8014..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$OnEmojiClickListener.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$ViewHolder$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$ViewHolder$1.dex
deleted file mode 100644
index 414a3af..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$ViewHolder$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$ViewHolder.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$ViewHolder.dex
deleted file mode 100644
index 83a1730..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter$ViewHolder.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter.dex
deleted file mode 100644
index 6560d31..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiAdapter.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiFragment.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiFragment.dex
deleted file mode 100644
index 0938d3e..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/EmojiFragment.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/GalleryUtils.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/GalleryUtils.dex
deleted file mode 100644
index 3e3349f..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/GalleryUtils.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$OnImageClickListener.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$OnImageClickListener.dex
deleted file mode 100644
index 8593cb9..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$OnImageClickListener.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$ViewHolder$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$ViewHolder$1.dex
deleted file mode 100644
index d3c71a4..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$ViewHolder$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$ViewHolder.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$ViewHolder.dex
deleted file mode 100644
index e8e2153..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter$ViewHolder.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter.dex
deleted file mode 100644
index 0c4ceb8..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageAdapter.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageFragment.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageFragment.dex
deleted file mode 100644
index db5a03c..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/ImageFragment.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/MainActivity.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/MainActivity.dex
deleted file mode 100644
index dc8db6c..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/MainActivity.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/MediaActivity.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/MediaActivity.dex
deleted file mode 100644
index 973e923..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/MediaActivity.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$1.dex
deleted file mode 100644
index 930be06..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$10.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$10.dex
deleted file mode 100644
index b5266ee..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$10.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$2.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$2.dex
deleted file mode 100644
index 58616e0..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$2.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$3.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$3.dex
deleted file mode 100644
index c293d77..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$3.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$4.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$4.dex
deleted file mode 100644
index 4f1067b..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$4.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$5.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$5.dex
deleted file mode 100644
index c876d07..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$5.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$6.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$6.dex
deleted file mode 100644
index 0d07984..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$6.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$7.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$7.dex
deleted file mode 100644
index 91bddcd..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$7.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$8.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$8.dex
deleted file mode 100644
index af44d34..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$8.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$9.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$9.dex
deleted file mode 100644
index a894e22..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$9.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$PreviewSlidePagerAdapter.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$PreviewSlidePagerAdapter.dex
deleted file mode 100644
index fe49036..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity$PreviewSlidePagerAdapter.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity.dex
deleted file mode 100644
index 5de4d5a..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/PhotoEditorActivity.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/UtilFunctions.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/UtilFunctions.dex
deleted file mode 100644
index f03ba18..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/UtilFunctions.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView$1.dex
deleted file mode 100644
index 9596d7e..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView$StopException.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView$StopException.dex
deleted file mode 100644
index 09c52ea..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView$StopException.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView.dex
deleted file mode 100644
index 9d51538..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/RealtimeBlurView.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$1.dex
deleted file mode 100644
index 145794e..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$2.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$2.dex
deleted file mode 100644
index d2057c2..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$2.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$DragHelperCallback.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$DragHelperCallback.dex
deleted file mode 100644
index 4fb8fd8..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$DragHelperCallback.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$LayoutParams.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$LayoutParams.dex
deleted file mode 100644
index e70095d..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$LayoutParams.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$PanelSlideListener.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$PanelSlideListener.dex
deleted file mode 100644
index b4bd0b9..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$PanelSlideListener.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$PanelState.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$PanelState.dex
deleted file mode 100644
index 72c80b2..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$PanelState.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SavedState$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SavedState$1.dex
deleted file mode 100644
index 319b8fd..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SavedState$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SavedState.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SavedState.dex
deleted file mode 100644
index 2a30441..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SavedState.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SimplePanelSlideListener.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SimplePanelSlideListener.dex
deleted file mode 100644
index 8679a80..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout$SimplePanelSlideListener.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout.dex
deleted file mode 100644
index 17b595a..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/SlidingUpPanelLayout.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$1.dex
deleted file mode 100644
index 9c62ee4..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$2.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$2.dex
deleted file mode 100644
index 8733d3c..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$2.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$Callback.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$Callback.dex
deleted file mode 100644
index c216258..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper$Callback.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper.dex
deleted file mode 100644
index dd3d5f0..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/com/ahmedadeltito/photoeditor/widget/ViewDragHelper.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/BuildConfig.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/BuildConfig.dex
deleted file mode 100644
index 7b6eb4b..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/BuildConfig.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorModule$1.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorModule$1.dex
deleted file mode 100644
index b51964e..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorModule$1.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorModule.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorModule.dex
deleted file mode 100644
index a649c3c..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorModule.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorPackage.dex b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorPackage.dex
deleted file mode 100644
index 570bcd0..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/debug/ui/photoeditor/RNPhotoEditorPackage.dex and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/desugar_graph.bin b/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/desugar_graph.bin
deleted file mode 100644
index 601f245..0000000
Binary files a/node_modules/react-native-photo-editor/android/build/.transforms/9fa9167a201ab5076f9ec2b26fa39bd1/desugar_graph.bin and /dev/null differ
diff --git a/node_modules/react-native-photo-editor/android/build/intermediates/aar_metadata/debug/aar-metadata.properties b/node_modules/react-native-photo-editor/android/build/intermediates/aar_metadata/debug/aar-metadata.properties
new file mode 100644
index 0000000..1211b1e
--- /dev/null
+++ b/node_modules/react-native-photo-editor/android/build/intermediates/aar_metadata/debug/aar-metadata.properties
@@ -0,0 +1,6 @@
+aarFormatVersion=1.0
+aarMetadataVersion=1.0
+minCompileSdk=1
+minCompileSdkExtension=0
+minAndroidGradlePluginVersion=1.0.0
+coreLibraryDesugaringEnabled=false
